### DISCUSSION TITLE
New compartment in MOBI - V and WT

### DISCUSSION THREAD

**Discussion by HLyster** [Open-Systems-Pharmacology/Forum#1174] [Models]
I have created an extra compartment (by cloning blood organ) in MOBI - but cant get the V and WT parameters to be added to the body weight and plasma volume, any help appreciated

**Comment by PavelBal** [comment]
Body weight is modeled as a sum of the weights of blood and tissue organs:

![image](https://user-images.githubusercontent.com/25903040/158562952-ffb52441-de04-4b41-977b-26622e4e176f.png)

If you examine the parameter `Weight of tissue organs`, you will see that this is the so-called "sum formula"

![image](https://user-images.githubusercontent.com/25903040/158563115-d5ac8dcb-8222-440a-82c4-d2b5db5abbce.png)

It sums up the values of all parameters called `Weight (tissue)`, that are in any organ that have a tag `TissueOrgan`. So you have the make sure that:

1) The organ (compartment?) you have added has the tag `TissueOrgan`

![image](https://user-images.githubusercontent.com/25903040/158563475-ef4918c1-cee3-46d6-8c89-48eb005bc4eb.png)

2) The organ has a parameter `Weight (tissue)`, see how it is defined in other organs:

![image](https://user-images.githubusercontent.com/25903040/158563623-7d0a3562-0dd8-461c-83ff-880ef8bdf839.png)


For plasma volume: you have to manually add the plasma volume of your new organ to the formula of the paramter `Volume (plasma)` located in `Organism`:

![image](https://user-images.githubusercontent.com/25903040/158563906-43197f37-2331-4a0a-b4f1-b78e6aa3df5f.png)

**Reply by HLyster** [reply]
Thank you - I am still struggling with adding WT - Volume worked OK

[Adding WT to ECMO compartment Github forum.docx](https://github.com/Open-Systems-Pharmacology/Forum/files/8717911/Adding.WT.to.ECMO.compartment.Github.forum.docx)

Any ideas? Or I have missed something?

Thanks

**Comment by PavelBal** [comment]
It is hard to figure out what is the problem without knowing what are the errors that are thrown when you create a simulation.

**Reply by HLyster** [reply]
Thank you - I will try and take screen shot - when it happens again - but as can see there is no wt calculated as part of the sum formula.. so dont understand why not
